content/docs/1.5/scalers/rabbitmq-queue.md

```diff
@@ -70,8 +97,19 @@ spec:
   - type: rabbitmq
     metadata:
       includeUnacked: "true"
-      # Required
-      apiHost: RabbitApiHost # references a value of format https://guest:password@localhost:443/vhostname
       queueName: testqueue
       queueLength: "20"
+    authenticationRef:
+      name: rabbitmq-consumer-trigger
+---
+apiVersion: keda.k8s.io/v1alpha1
```

Comment:
Let's create this before the scaledobject for logical ordering

Reply:
Done